CMakeLists.txt

```diff
@@ -485,10 +489,19 @@ if(MIOPEN_USE_ROCBLAS)
     set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocblas")
 endif()
 
+# PR #2785 MIOpenDriver to use rocrand to init buffers
+set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocrand")
+
```

Comment:
@junliume This is duplicate of lines 499-504, but not protected with if(MIOPEN_BUILD_DRIVER).

Reply:
@atamazov yes this line should be removed, I forgot to update the branch.